terraform_compliance/common/helper.py

```diff
@@ -257,7 +257,7 @@ def search_regex_in_list(regex, target_list):
 
 def seek_value_in_dict(needle, haystack, address=None):
     findings = []
-    if isinstance(haystack, (str, int, bool, float)) and needle in haystack:
+    if isinstance(haystack, (str, int, bool, float)) and needle in str(haystack):
```

Comment:
if we convert `haystack` to `string`, doesn't `needle` should also be a `string` as well here ?